src/System.ComponentModel.Primitives/src/System/ComponentModel/BrowsableAttribute.cs

```diff
@@ -0,0 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.ComponentModel
+{
+    /// <devdoc>
```

Comment:
With this code coming in new, could we avoid the ```<devdoc>```s and just use normal XML comments, e.g. ```<summary>``` here?